packages/cli/src/argparse.ts

```diff
@@ -1380,6 +1380,33 @@ export const CLI_ARGS = {
         '\n',
       group: 'Key Management',
     },
+    migrate_subdomains: {
```

Comment:
Could you add a paramater `registrar` similar to in [`register_subdomain`](https://docs.hiro.so/references/stacks-cli#register_subdomain) ? this would make the beta url easier to test: https://github.com/stacks-network/subdomain-registrar/pull/88#issuecomment-1154010940 and also allow users to use a custom registrar without rebuilding.